src/logger/connectivityTask.c

```diff
@@ -521,149 +521,109 @@ void cellular_buffering_task(void *params)
                                 }
-
-                                if (fs_good) {
-                                        bool sd_write_validated = test_sd(NULL, 1, 0, 1);
-                                        if (sd_write_validated) {
-                                                FRESULT fopen_rc = f_open(cellular_state.buffer_file, TELEMETRY_BUFFER_FILENAME, FA_READ | FA_WRITE | FA_CREATE_ALWAYS);
-                                                if (FR_OK == fopen_rc) {
-                                                        FRESULT truncate_rc = f_truncate(cellular_state.buffer_file);
-                                                        if (FR_OK == truncate_rc){
-                                                                cellular_state.buffer_file_open = true;
-                                                                /* try to connect immediately on the first re-attempt*/
-                                                                re_open_buffer_file_timeout = 0;
-                                                                buffer_file_open_retries = 0;
-                                                        }
-                                                        else{
-                                                                if (!buffer_file_open_retries)
-                                                                        pr_info_int_msg(_LOG_PFX "Error truncating telemetry buffer file: ", truncate_rc);
-                                                        }
-                                                }
-                                                else {
-                                                        if (!buffer_file_open_retries)
-                                                                pr_info_int_msg(_LOG_PFX "Error opening telemetry buffer file: ", fopen_rc);
-                                                }
-                                        }
-                                        else {
-                                                if (!buffer_file_open_retries)
-                                                        pr_info(_LOG_PFX "SD write verification failed. Try re-formatting SD card\r\n");
-                                        }
-                                }
-                                fs_unlock();
-                                if (!cellular_state.buffer_file_open) {
-                                        /* try re-opening a bit later */
-                                        re_open_buffer_file_timeout = TELEMETRY_BUFFER_FILE_RETRY_MS;
-                                        buffer_file_open_retries++;
-                                }
-                                pr_debug_str_msg(_LOG_PFX "Creating telemetry buffer file: ", cellular_state.buffer_file_open ? "win" : "fail");
                         }
+                        else
+                        {
+                                vTaskDelay( 1000 );
```

Comment:
@brentpicasso not happy with this, but I wasn't sure how frequently this was being invoked on failure.